packages/theme/css/characteristics/text.css

```diff
@@ -24,12 +24,12 @@
   /* H3 */
   --uitk-text-h3-fontWeight: var(--uitk-typography-fontWeight-semiBold);
   --uitk-text-h3-fontWeight-small: var(--uitk-typography-fontWeight-regular);
-  --uitk-text-h3-fontWeight-strong: var(--uitk-typography-fontWeight-extraBold);
+  --uitk-text-h3-fontWeight-strong: var(--uitk-typography-fontWeight-bold);
```

Comment:
Are these changes intentional? If so they should be in a separate PR

Reply:
yes, was planning to :)